scipy/special/cdf_wrappers.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include "cdf_wrappers.h"
+#include "cephes.h"
```

Comment:
`cdflib` also has a `stdtr` and `stdtri` implementation but SciPy uses the cephes `implementation` so prefer it instead.

Reply:
Cephes is already linked for the extensions where `cdf_wrappers.c` is used in `setup.py`, so this seems fine to me.

Reply:
Thanks for taking a look. I think this PR is ready from my side!

Reply:
CI is very unhappy, can you rebase so we pick up the fixes in master?

Reply:
Done.